packages/@storefront/flux-capacitor/src/core/selectors.ts

```diff
@@ -363,6 +363,21 @@ namespace Selectors {
   export const config = (state: Store.State) =>
     state.session.config;
 
+  /**
+   * If called with a `value` parameter, returns a boolean value indicating the
+   * current fetching state otherwise returns the full `isFetching` object.
+   * @param value Accepted values conform to the [[Store.IsFetching]] properties.
```

Comment:
Needs an `@return` line.

Reply:
No, it's auto-generated

Reply:
We just added to the auto-generated param definition bc it doesn't link the type

Reply:
It's more for the English explanation for what it returns.

Reply:
Ok but we don't need it, so no